api_tests/e2e/rfc003/eth_btc/refund.ts

```diff
@@ -0,0 +1,41 @@
+import { Actor } from "../../../lib_sdk/actors/actor";
+import { AssetKind } from "../../../lib_sdk/asset";
+import { createActors } from "../../../lib_sdk/create_actors";
+
+setTimeout(function() {
+    let alice: Actor;
+    let bob: Actor;
+
+    beforeEach(async function() {
+        this.timeout(20000);
+        const actors = await createActors("e2e-rfc003-eth-btc-refund.log");
+        alice = actors.alice;
+        bob = actors.bob;
+    });
+
+    afterEach(() => {
+        if (alice) {
+            alice.stop();
+        }
+        if (bob) {
+            bob.stop();
+        }
+    });
+
+    describe("Bob can refund and then Alice refunds", function() {
+        this.timeout(60000);
+        it("ether bitcoin", async function() {
+            await alice.sendRequest(AssetKind.Ether, AssetKind.Bitcoin);
+            await bob.accept();
+
+            await alice.fund();
+            await bob.fund();
+            await bob.refund();
+            await alice.refund();
+
+            await bob.assertRefunded();
```

Comment:
If I understand correctly you already do this assertion in the `Actor` class.

Reply:
Not quite. 
* In `alice/bob.refund()` we assert cnd nodes moved to the correct state
* In `alice/bob.assertRefunded()` we assert that the value was actually moved back to their wallets. 

One could probably merge those two functions.

Reply:
ok cool :)